packages/devtools_server/docs/daemon.md

```diff
@@ -69,9 +69,11 @@ This request is only used for testing purposes so is currently "undocumented"
 This request lists all DevTools instances that are currently connected back to the server along with which VM services they're connected to and the pages they are showing. The request requires no `params`.
 -->
 
-### launchDevTools Request (VM Service)
+### devTools.launch Request / launchDevTools VM Service
```

Comment:
You might want to specialize the docs for these two calls (or, doc one, and then doc the differences for the 2nd).

Reply:
Done! (04f1ac54)